src/api/index.js

```diff
@@ -0,0 +1,272 @@
+import config from '../config'
+import {mergeRecursive} from "../helpers/mergeObjects";
+import redirect from "react-router/es/Redirect";
+
+
+class ApiClass {
+    constructor() {
+        this.url = `${config.apiUrl}`;
+    }
+
+    makeRequest = ({url, data, token}) => {
+        const _data = {
+            headers: {
+                'Accept': 'application/json',
+                'Content-Type': 'application/json',
+                ...!!token && {Authorization: `Bearer ${token}`},
+
+            },
+            credentials: 'include',
+
+        };
+
+        const mergedData = mergeRecursive(data, _data);
+
+        return fetch(url, mergedData)
+            .then(response => {
+                if (response.status <= 300) {
+                    return response;
+                }
+                console.log(response.status)
+                if (response.status === 302) {
+                    redirect(response.headers.location);
```

Comment:
after this redirect code execution will go ahead.
Are you sure that you need to execute code after this line?